build.sbt

```diff
@@ -0,0 +1,36 @@
+import Dependencies._
+
+scalaVersion := "2.13.4"
+
+lazy val commonSettings = Seq(
+  resolvers ++= Seq[Resolver](
+    "notify-java-client" at "https://dl.bintray.com/gov-uk-notify/maven/"
+  )
+)
+
+lazy val notifySpi = (project in file("govuk-notify-spi"))
+  .settings(
+    name := "govuk-notify-spi",
+    commonSettings,
+    scalaVersion := "2.13.4",
+    libraryDependencies ++= Seq(
+      keycloakCore,
+      keycloakModelJpa,
+      keycloakServerSpi,
+      mockito % Test,
+      notifyJavaClient,
+      scalaTest % Test,
+      typeSafeConfig
+    ),
+    assemblyJarName in assembly := "govuk-notify-spi.jar",
+    fork in Test := true,
```

Comment:
What is this line used for? What happens if it's not there?

Reply:
Makes sure the environment variables for running the tests are not the ones set on the machine where the test is run, when using sbt to run the tests: https://stackoverflow.com/questions/44298847/why-do-we-need-to-add-fork-in-run-true-when-running-spark-sbt-application

Reply:
Huh. I hadn't noticed this in [some of the other projects](https://github.com/search?q=org%3Anationalarchives+%22fork+in+Test%22&type=code). I would have thought that makes it difficult to genuinely set environment variables for the tests, but I guess we can cross that bridge when we come to it.